app/platform/fabric/utils/FabricConst.ts

```diff
@@ -18,3 +18,8 @@ export const fabric = {
 		NOTITY_TYPE_CLIENTERROR: '6'
 	}
 };
+
+export enum PurgeModes {
+	'TIME',
```

Comment:
If we have the NONE as purge mode, then we need to handle that case, can you check this case once.